ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -749,6 +751,9 @@ public class RouterConfig {
   @Default("false")
   public final boolean routerReservedMetadataEnabled;
 
+  @Config(ROUTER_GET_OPERATION_DEPRIORITIZE_BOOTSTRAP_REPLICAS)
```

Comment:
Do we really need the config. No harm in deprioritizing the bootstrap replica by default.

Reply:
just want to have a layer of protection if this logic is somehow wrong.